### DISCUSSION TITLE
Official list of plugins

### DISCUSSION THREAD

**Discussion by Ortovoxx** [payloadcms/payload#1152] [Feature Requests & Ideas]
Is there currently an official list of payload plugins? If there is it's very well hidden.

I think it would be great to compile an official list of great plugins for payload. Both officially created plugins and community created. Users who develop their own plugins can request for theirs to be added to the list.

Adding this list to the readme would work well in my opinion.

**Comment by DanRibbens** [comment]
The short answer here is to use the link https://github.com/topics/payload-plugin
.
We ask plugin authors to add the topic "payload-plugin" to help with discovery. The long term plan is to have a dedicated page on the Payload website.

**Reply by Ortovoxx** [reply]
That's exactly what I was looking for. Perfect thank you! 

Could you link where it says you should upload plugins to that topic?

**Reply by DanRibbens** [reply]
At the bottom of the plugins page we talk about the topic.

https://payloadcms.com/docs/plugins/overview#available-plugins
